config.json

```diff
@@ -1102,6 +1102,19 @@
       "unlocked_by": "scrabble-score",
       "uuid": "2e760ae2-fadd-4d31-9639-c4554e2826e9"
     },
+    {
+      "core": false,
+      "difficulty": 7,
+      "slug": "dominoes",
+      "topics": [
+        "games",
+        "algorithms",
+        "lists",
+        "recursion"
```

Comment:
Do you have to use recursion to solve this exercise? If not, then I would probably not add it to the topics. The topics should be suggestions of things the user might or must use to solve the exercise. It's normally better to use iteration than recursion in Java since it's more suitable to the language. So I would only add recusion to the topics if the exercise has to, or definitely should, be solved using recursion. Otherwise it can be seen as encouraging the users to implement a solution which isn't necessarily suitable for Java 🙂

Reply:
Is it ok that I use recursion in the .meta reference solution?

Can you think of any other topics I could/should include?

Reply:
Solving the problem any way you see fit is completely fine :slightly_smiling_face: 

We should however refrain from using labels that aren't a core part of the exercise

Reply:
Yes, I agree with @sjwarner-bp, you can solve the exercise any way you want 🙂 That won't get shown to the users, it's just a way for us to check that the tests work 🙂 

If you decide to throw exceptions in your solution and expect exceptions in the tests then you could add `exception_handling` to the topics. If you decide to go with optionals then you could add `optional_values` to the topics 🙂 Apart from that I think the topics you have are a good selection 🙂